src/syrupy/__init__.py

```diff
@@ -75,6 +92,7 @@ def pytest_sessionstart(session: Any) -> None:
     session._syrupy = SnapshotSession(
         warn_unused_snapshots=config.option.warn_unused_snapshots,
         update_snapshots=config.option.update_snapshots,
+        default_extension_class=config.option.default_extension,
```

Comment:
(nb) is there an advantage to storing the default in the session? We reference the request.config directly in the snapshot fixture on line 139.